index.ts

```diff
@@ -259,11 +259,10 @@ async function init() {
       text({
         message: language.projectName.message,
         placeholder: defaultProjectName,
-        validate: (value) =>
-          value.trim().length > 0 ? undefined : language.projectName.invalidMessage,
```

Comment:
Don't remove the `validate` function, we still need it to reject inputs with whitespaces only, like `  `.

Reply:
I'm okay with converting space-only inputs to `vue-project`, but the problem with the implementation is that the feedback isn't as clear:
Hit Enter directly, `vue-project` is displayed as the answer to the prompt:
<img width="519" alt="image" src="https://github.com/user-attachments/assets/4f3ce14d-d552-4cd9-995b-06091fa8e76b" />

Use two spaces, we still use `vue-project` as the project name but the answer on screen is empty:
<img width="519" alt="image" src="https://github.com/user-attachments/assets/f9b355de-c172-4ea8-9bd2-3a52be3c2baa" />

Reply:
The easiest fix I found is to accept `''` but reject `'  '`.
But feel free to propose any other way to work around this issue.

Reply:
That makes sense. At the moment, `The easiest fix I found is to accept '' but reject '  '` seems to be the simplest and most reasonable modification. 👍